src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,40 @@
+import { Todo } from '../../types/Todo';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+type Props = {
+  todos: Todo[],
+  onDeleteTodo: (id: number) => void,
+  tempTodo: Todo | null,
+  loadingTodoId: number | null,
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos,
+  onDeleteTodo = () => { },
```

Comment:
Providing a default implementation for onDeleteTodo as en empty function is not a good practice. It should be a required prop